PathManager/Src/PM_Interface.cpp

```diff
@@ -0,0 +1,22 @@
+#include "PM_Interface.h"
+#include "PM.hpp"
+
+static pathManager pathMng;
+PathMan::_Path_Manager_Cycle_Status pathManStatus;
+
+
+bool PathManagerInterfaceExecute(void) {
```

Comment:
Should there be a waitUntil in here to ensure that the operation is at a set frequency?